.github/workflows/test.yml

```diff
@@ -0,0 +1,43 @@
+name: Test
+
+on:
+  push:
+  pull_request:
+  schedule:
+    # Monthly, chosen randomly to avoid peak times
+    - cron:  '5 15 14 * *'
```

Comment:
Why is it configured to run on a schedule?

Reply:
Because Travis has a cron too:

![image](https://user-images.githubusercontent.com/1324225/102066219-3b427580-3e02-11eb-8a32-210c6f760d0d.png)

Although we forgot to update that when we did `master` -> `main` and it's not run for 5 months :)

https://travis-ci.org/github/sb2nov/mac-setup/settings

Reply:
Oh. I think we can disable it? I don't see the value. WDYT?

Reply:
It can be useful for tracking down failures when dependencies break, with no change in the repo. I guess less useful for this one than other projects?

But also just spotted they're disabled:

> Warning: To prevent unnecessary workflow runs, scheduled workflows may be disabled automatically. When a public repository is forked, scheduled workflows are disabled by default. In a public repository, scheduled workflows are automatically disabled when no repository activity has occurred in 60 days.

https://docs.github.com/en/free-pro-team@latest/actions/managing-workflow-runs/disabling-and-enabling-a-workflow

Shall we remove the cron?

Reply:
I vote for removing. Our dependency versions are pinned and because we're serving statically generated content it doesn't matter. Yay for simple projects! :D

Reply:
Removed, and make the config simpler here too!